templates/events/event-edit.html

```diff
@@ -187,17 +187,23 @@ <h2><a class="article-title">Edit Event</a></h2>
             <div class="col-12 col-sm-2">
               <h5>Group<span style="color: red">*</span> </h5>
             </div>
+
             <div class="col-sm-9">
-              <div class="form-group">
-                  <select name="createdByGroupId" id="createdByGroupId" class="form-control w-auto" title="Choose a group..." data-width="auto" required>
+              <div class="user-group-menu eventsmanager-dropdown-menu">
                   {% for group in groups %}
                       {% if post.createdByGroupId == group['id'] %}
-                      <option value="{{group['id']}}" id={{group['id']}} selected>{{group['title']}}</option>
-                      {% else %}
-                      <option value="{{group['id']}}" id={{group['id']}}>{{group['title']}}</option>
+                          <input type="text" name="createdByGroupTitle" id="createdByGroupTitle" value="{{group['title']}}" class="form-control dropdown-menu-inputfield" placeholder="{{group['title']}}" readonly>
+                          <input type="hidden" name="createdByGroupId" id="createdByGroupId" value="{{group['id']}}">
                       {% endif %}
                   {% endfor %}
-                  </select>
+                  <div class="dropdown-line" style="width:100%;height:2px;border:1px solid #e84a27;background: #e84a27;border-width: 1px;"></div>
+                  <div class="dropdown-menu-option">
+                      {% for group in groups %}
+{#                        {% if post.createdByGroupId != group['id'] %}#}
```

Comment:
Please remove the commented-out code. Similarly, at another place in the PR. Thanks.

Reply:
fixed